Dockerfile.muo

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/go-toolset:1.16.12 AS builder
+FROM arointsvc.azurecr.io/ubi8/go-toolset:1.16.12 AS builder
```

Comment:
Can this be bumped to 1.17.7? Then we'd only need to mirror the one go-toolset

Reply:
@s-amann both tags are already mirrored and available to pull from `arointsvc`. I am not sure why we use 1.16.12 instead of 1.17.7 here. Maybe @hawkowl knows?